tools/packaging/deb/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Base dockerfile containing ubuntu and istio debian.
 # Can be used for testing
+# hadolint ignore=DL3006
 FROM istionightly/base_debug
```

Comment:
we need to be tagging istionightly builds imo.

Reply:
Yes we do, this might be our main source of version tracking problem

Reply:
I totally agree. We need to start tagging this to a version since this is used as the base image for many of our Istio components.